sdks/python/src/opik/integrations/bedrock/chunks_aggregator.py

```diff
@@ -35,3 +35,15 @@ def aggregate(items: List[Dict[str, Any]]) -> Dict[str, Any]:
                 result["metrics"]["latencyMs"] = metadata["metrics"]["latencyMs"]
 
     return result
+
+
+def aggregate_invoke_agent_chunks(items: List[Dict[str, Any]]) -> Dict[str, Any]:
+    merged_chunks = b""
+
+    for item in items:
+        if "chunk" in item:
+            merged_chunks += item["chunk"]["bytes"]
+
+    result: Dict[str, Any] = {"output": merged_chunks.decode("utf-8")}
```

Comment:
It would be nice to track not only the output.
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/bedrock-agent-runtime/client/invoke_agent.html take a look at response format

I can see that there is also usage information. Pay attention that the agent has multiple steps of foundation model invocation - preprocessing, postprocessing, orchestration, each of them has their own usage.